cli/dts/lib.deno.unstable.d.ts

```diff
@@ -155,7 +155,7 @@ declare namespace Deno {
   }
 
   export interface DiagnosticMessageChain {
-    message: string;
+    messageText: string;
```

Comment:
This is correct https://github.com/microsoft/TypeScript/blob/8362a0f929d74ff46828016ec67c05744a8dbb3c/lib/typescript.d.ts#L2772